docs/Config.md

```diff
@@ -4,13 +4,16 @@
 
 The location of the user config will differ depending on your OS. You can open it via lazydocker by opening the application, clicking on the 'project' panel at the top left and pressing 'o' (or pressing 'e' if your files open in vim).
 
-Changes to the user config will only take place after closing and re-opening lazydocker
+Changes to the user config will only take place after closing and re-opening lazydocker.
 
 ### Locations:
 
 - OSX: `~/Library/Application Support/jesseduffield/lazydocker/config.yml`
 - Linux: `~/.config/lazydocker/config.yml`
 - Windows: `C:\\Users\\<User>\\AppData\\Roaming\\jesseduffield\\lazydocker\\config.yml` (I think)
+- UNIX when `XDG_CONFIG_HOME` is found: `$XDG_CONFIG_HOME/lazydocker/config.yml`
```

Comment:
Readers may stop reading once they see OSX and Linux. We should say on each of those lines '(if XDF_CONFIG_HOME env variable is not set)' and we should have the unix line first.

Reply:
Alright i've changed the wording, let me know what you think